src/router/index.js

```diff
@@ -107,7 +109,7 @@ const router = new Router({
           path: 'events/:eventId/confirmation',
           name: 'EventBookingConfirmation',
           component: BookingConfirmation,
-          beforeEnter: loggedInNavGuard,
+          beforeEnter: MultiGuard([orderExistsNavGuard, loggedInNavGuard]),
```

Comment:
Order should be swapped around. Check if they're logged in before trying for the order. Otherwise we could end up requesting the order for user `undefined`